wiki/Wiki entry - Peter Gollai.txt

```diff
@@ -0,0 +1,41 @@
+
+Q: 1. Since the study has been made by a small sample of only 18 journalists coming from the USA, you can’t draw any conclusions regarding the behaviour and perspective of international journalism. For this reason a much bigger sample from international journalists could be made. 
+
+2. Because the authors chose to do interviews as their method of answering their research question, the results can differ from the actual reality. Hence, a content analysis should be made to confirm or deny the given statements that the journalists made in order to get a better understanding of this topic.
+
+
+
+
+
+
+
+
+
+
+
+
+Summary:
+In order to answer the research questions the authors did quantitative, semi-structured interviews with 18 professional journalists, all living in the United States. They searched participants through LinkedIn, snowball sampling and suggestions from participants who already completed the interviews. They wanted to answer two questions. First, What algorithmic ‘folk theories’ permeate journalistic practice and second, how and to what extent do journalists perceive these algorithmic ‘folk theories’ to influence their editorial decision making and gatekeeping practices at various levels. The interviews have been conducted between August 2019 and February 2020 and they lasted on average about 65 minutes. 
+
+Regarding the first research question, 83,33% (15 of 18) have said that they are aware of algorithms on social media platforms and thought about their mechanism. Furthermore, most of them understood them as filters that do or do not allow users to see their posts. But, the elements that do or do not boost your posts varied a lot between the interviewees, especially in the topics engagement, publisher attributes and the used platform. 
+
+Focusing on research question two, 33,3% said that they don’t think about algorithms when creating a post, which can be explained by either their organization handles social media as a minor matter or that themselves are not involved in social media posts directly. Only two participants had been explicitly asked to not post something because it wouldn’t do well on social media.
+
+In conclusion, the results don’t show that algorithms have a huge impact on editorial decision-making in modern-day journalism. Moreover, algorithms are becoming less important to journalists because they decide to redirect their focus to search engine optimization and content aggregators. 
+
+Nevertheless, journalists cannot disregard the importance that algorithms have on their posted content that creates pressure to post something likeable, leading to restricted autonomy for the journalist and the user.
```

Comment:
Does the study provide any insights about how this might relate to individuals' autonomy need experiences?